src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1022,6 +1022,17 @@ public void deleteStudent(String courseId, String studentEmail) {
         studentsLogic.deleteStudentCascade(courseId, studentEmail);
     }
 
+    /**
+     * Deletes all the students in the course.
+     *
+     * @param courseId course id for the students
+     */
+    public void deleteAllStudentsInCourse(String courseId) {
+
+        Assumption.assertNotNull(courseId);
+        studentsLogic.deleteAllStudentsInCourse(courseId);
```

Comment:
I am working on this to find out the efficient way. @whipermr5 I wanted to visualize DB on some GUI tool but unable to find something to see NOSQL data. Usually database is stored in file `local_db.bin` and I want to see the data on some tool. Any suggestions?